packages/helium-cli/src/create-dao.ts

```diff
@@ -103,41 +184,232 @@ async function run() {
     metadataUrl: `${argv.bucket}/dc.json`,
   });
 
+  await createAndMint({
+    provider,
+    mintKeypair: councilKeypair,
+    amount: argv.numCouncil,
+    decimals: 0,
+    metadataUrl: `${argv.bucket}/council.json`,
+    to: councilWallet,
+  });
+
+  let instructions: TransactionInstruction[] = [];
+  const govProgramVersion = await getGovernanceProgramVersion(
+    conn,
+    govProgramId
+  );
+
+  const realmName = argv.realmName;
+  const realm = await PublicKey.findProgramAddressSync(
+    [Buffer.from("governance", "utf-8"), Buffer.from(realmName, "utf-8")],
+    govProgramId
+  )[0];
+  if (!(await exists(conn, realm))) {
+    console.log("Initializing Realm");
+    await withCreateRealm(
+      instructions,
+      govProgramId,
+      govProgramVersion,
+      realmName,
+      provider.wallet.publicKey, // realmAuthorityPk
+      hntKeypair.publicKey, // communityMintPk
+      provider.wallet.publicKey, // payer
+      councilKeypair.publicKey, // councilMintPk
+      MintMaxVoteWeightSource.FULL_SUPPLY_FRACTION,
+      new anchor.BN(1), // minCommunityWeightToCreateGovernance
```

Comment:
We might want to change this? This means I need 1 weight to create proposals I think. Idk if we actually want community creating proposals or if they're just allocated into realms via the council?